### ISSUE TITLE
Plot_transcripts error with STAR but not SOAP imports. 

### ISSUE THREAD

**Issue by ahdee** [stianlagstad/chimeraviz#46]
Hi, 
I'm using STAR-Fusion-v1.3.2 and the chimeraviz:  "1.6.0".   I was able to input the tsv file from star-fusion and circos plot can be generated.  However, when I tried

```
plot_transcripts(
    fusion = get_fusion_by_id(star.fusion, 1) ,
    edb = edb, 
    reduce_transcripts = T)
```
I get this error. 
```
Fetching transcripts for gene partners..
Error in get_transcripts_ensembl_db(fusion, edb) : 
  No transcripts available for the genes ZNF384 and TAF15.
```
When I the same fusion and sample is imported with soap the code runs perfectly.  
I included a test tsvfrom Star.   ( file name is .txt so that I can upload example) 

thanks! 

[test.txt](https://github.com/stianlagstad/chimeraviz/files/2156151/test.txt)

**Comment by stianlagstad** [comment]
Hi @ahdee! I'm not able to look at this right now but could you also share your SOAPFuse results file? Then I can look at both and compare when I have the time :)

**Comment by sraorao** [comment]
Hi @ahdee and @stianlagstad, I encountered the same problem, and fyi I think the problem is with the ensembl ids as generated by STAR-Fusion which includes the version numbers (following the period). Stripping the version number manually seems to work.

**Comment by stianlagstad** [comment]
Thank you very much, @sraorao!:) @ahdee please let us know whether this works for you. If it does, I can modify the import function for STAR-Fusion to strip the version number.

**Comment by ahdee** [comment]
Yes it works!  Had to manually delete the ENS...\\.+

**Comment by stianlagstad** [comment]
Thank you very much, @ahdee and @sraorao. I've fixed the parsing of the Starfusion ensembl id, but unfortunately not in time to update the Bioconductor 3.7 release. You can however get the fix by cloning the  3.7 branch from GitHub: https://github.com/stianlagstad/chimeraviz/tree/RELEASE_3_7.

The fix will be present in the upcoming 3.8 release :)

3.7 fix: https://github.com/stianlagstad/chimeraviz/pull/50
3.8 fix: https://github.com/stianlagstad/chimeraviz/pull/49
